azure-functions/src/http/response_builder.rs

```diff
@@ -19,9 +19,9 @@ impl ResponseBuilder {
     /// use azure_functions::bindings::HttpResponse;
     /// use azure_functions::http::Status;
     ///
-    /// let response: HttpResponse = HttpResponse::build()
```

Comment:
@peterhuene, with the reversion from `SendGridMessage` to `MessageBuilder` in `message_builder`. Should I also change `HttpResponse::build...` to `ResponseBuilder::new`?